packages/cozy-harvest-lib/src/models/ConnectionFlow.js

```diff
@@ -474,12 +475,42 @@ export class ConnectionFlow {
       konnector,
       t
     })
+
+    await this.addDefaultFolderPath(client, {
+      trigger: ensuredTrigger,
+      account,
+      konnector
+    })
+
     logger.info(`Trigger is ${ensuredTrigger._id}`)
     this.trigger = ensuredTrigger
     this.emit(UPDATE_EVENT)
     await this.launch()
   }
 
+  /**
+   * Add a default folder path to the account if any folder is needed by the connector
+   * and/or the account does not have it yet.
+   * The account is saved if any change is done to it.
+   * This defaultFolderPath is needed by the stack te recreate the destination folder this
+   * had beed removed by any mean : drive application, desktop application etc
+   *
+   * @param  {CozyClient} client - A cozy client
+   * @param  {io.cozy.triggers} options.trigger
+   * @param  {io.cozy.accounts} options.account
+   * @param  {io.cozy.konnector} options.konnector
+   */
```

Comment:
nit: I don't understand why Doctypes are singular and plural:
-  io.cozy.triggers => plural
- io.cozy.accounts => plural
- io.cozy.konnector => singular

Reply:
Simple mistake. The good one is plural